src/extras/MovieClip.js

```diff
@@ -170,8 +203,16 @@ MovieClip.prototype.gotoAndStop = function (frameNumber)
 
     this._currentTime = frameNumber;
 
-    var round = Math.floor(this._currentTime);
-    this._texture = this._textures[round % this._textures.length];
+    if (this._durations !== null)
```

Comment:
Is this needed at gotoAndStop? Feel like this should just click over to the texture immediately and be done.